app/assets/javascripts/thin_man.js

```diff
@@ -47,7 +47,13 @@ var initThinMan = function(){
       getTarget: function(){
         var target_selector = this.jq_obj.data('ajax-target');
         if(target_selector){
-          this.target = $(target_selector);
+          if($(target_selector).length > 0){
+            this.target = $(target_selector);
+          }else{
+            console.log('Warning! Selector ' + target_selector + ' not found on page')
```

Comment:
@gomayonqui, I just thought of something. A programmer might not know which library is warning about the target. I think the warning should have 'Thin Man' in it somewhere so the programmer knows to look at ajax stuff for the missing target.